dask/dataframe/io/tests/test_parquet.py

```diff
@@ -938,6 +938,56 @@ def test_to_parquet_default_writes_nulls(tmpdir):
     assert table[1].null_count == 2
 
 
+def test_to_parquet_consistent_schema_partition_pyarrow(tmpdir, engine):
```

Comment:
@mmccarty this test passes without any code change in `write_partition`. That is, it's not a failing test before code modification.
 
Was your thinking with this test just to add more coverage? 

Just wanted to be sure as I haven't deeply thought this through i just copy-pasted your tests.